cms/templates/js/container-unit-access.underscore

```diff
@@ -0,0 +1,15 @@
+<%
+var selectedGroupsLabel = userPartitionInfo['selected_groups_label'];
+var selectedPartitionIndex = userPartitionInfo['selected_partition_index'];
+%>
+<% if (selectedGroupsLabel) { %>
+    <small class="access-message"><%-
```

Comment:
I have never seen the "small" HTML tag before. We always use CSS to control display. @roderickmorales is this OK to use?